pkg/volume/glusterfs/glusterfs.go

```diff
@@ -966,6 +969,11 @@ func parseClassParameters(params map[string]string, kubeClient clientset.Interfa
 		case "volumetype":
 			parseVolumeType = v
 
+		case "volumeoptions":
+			if len(v) != 0 {
+				parseVolumeOptions = v
```

Comment:
@rootfs no. If the option specified is wrong, the volume creation will fail and it will be logged in heketi as like any other volume creation failure from glusterd. Because options are tried as part of volume creation in heketi.